e2etest/local_transfer_test.go

```diff
@@ -683,8 +672,8 @@ func TestChainIDWithJutland(t *testing.T) {
 		require.NoError(t, err)
 		err = bc.CommitBlock(blk)
 		require.NoError(t, err)
-		require.Equal(t, c.success, len(blk.Actions))
-		require.Equal(t, c.success, len(blk.Receipts))
+		require.Equal(t, 1, len(blk.Actions))
```

Comment:
for last test case {2, false}, it should be `len(blk.Actions) == 0`, since the tx will be rejected.

i found the reason is b/c the `continue` in workingset.go, it is actually needed (to skip the for loop outside of switch), so let's add it back

on the other hand, the test here is to allow us find such mistake, should not just change to `1` and let test pass